### PR TITLE
Add Vietnamese channels

### PR DESCRIPTION
**Pull request by freearhey**
Added links from https://github.com/iptv-org/iptv/issues/2731#issuecomment-834531656

### CODE REVIEW

File: channels/vn.m3u [hunk 1]
```diff
@@ -29,3 +29,75 @@ http://thtg.vn:8001/thtg.m3u8
...
+#EXTINF:-1 tvg-id="TTV.vn" tvg-name="TTV" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="" group-title="",TTV (406p)
+https://truyenhinh.vnptvas.vn/live.m3u8?c=vstv208&q=high&type=tv&token=1111&gcId=1532&userId=&deviceId=&deviceType=&location=NA&pkg=pkg11
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="" group-title="",Oh Muang Lao TV
+https://livefta.malimarcdn.com/ftaedge00/ohmuanglao.stream/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="" group-title="",Tea TV
+https://livefta.malimarcdn.com/ftaedge00/teatv.sdp/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="" group-title="",Good Idea TV
+https://livefta.malimarcdn.com/ftaedge00/gooditv.sdp/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="" group-title="",Lao Muslim TV
+https://livefta.malimarcdn.com/ftaedge00/ddtvlao.stream/playlist.m3u8
```
Comment (LaneSh4d0w) [inline channels/vn.m3u:41]: vlcsnap-2021-05-31-17h27m28s816
Not 24/7

File: channels/vn.m3u [hunk 2]
```diff
@@ -29,3 +29,75 @@ http://thtg.vn:8001/thtg.m3u8
...
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="" group-title="",Houng Fa
+https://livefta.malimarcdn.com/ftaedge00/houngfa.sdp/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="" group-title="",Dhammasapha TV
+https://livefta.malimarcdn.com/ftaedge00/dhammasapha.stream/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="" group-title="",Khomsanh TV
+https://livefta.malimarcdn.com/ftaedge00/lstv2.sdp/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="" group-title="",Lao Heritage Foundation TV
+https://livefta.malimarcdn.com/ftaedge00/laoheritagetv.sdp/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="" group-title="",Supreme Master TV
+https://lbs-us1.suprememastertv.com/720p.m3u8
```
Comment (LaneSh4d0w) [inline channels/vn.m3u:63]: Language of this channel is not only Vietnamese, I see some English parts.
vlcsnap-2021-05-31-17h29m40s270

File: channels/vn.m3u [hunk 3]
```diff
@@ -29,3 +29,75 @@ http://thtg.vn:8001/thtg.m3u8
...
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="" group-title="",LS TV
+https://livefta.malimarcdn.com/ftaedge00/laostv.sdp/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="" group-title="",Look Thoong TV
+https://livefta.malimarcdn.com/ftaedge00/lookthoongtv.sdp/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="" group-title="",Uniquely Thai
+https://livefta.malimarcdn.com/ftaedge00/uniquely.sdp/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="" group-title="",Vanphenh TV
+https://livefta.malimarcdn.com/ftaedge00/vanphenhtv.sdp/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="" group-title="",Sieng Khaen Lao TV
+https://livefta.malimarcdn.com/ftaedge00/vlctemple.sdp/playlist.m3u8
```
Comment (LaneSh4d0w) [inline channels/vn.m3u:81]: Black screen on this one, probably not 24/7
vlcsnap-2021-05-31-17h32m12s247

File: channels/vn.m3u [hunk 4]
```diff
@@ -29,3 +29,75 @@ http://thtg.vn:8001/thtg.m3u8
...
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="" group-title="",Lao Champa TV 1
+https://livefta.malimarcdn.com/ftaedge00/laochampa1.sdp/playlist.m3u8
```
Comment (LaneSh4d0w) [inline channels/vn.m3u:83]: Language of this channel is not only Vietnamese, I see some English parts.
vlcsnap-2021-05-31-17h33m34s132

File: channels/vn.m3u [hunk 5]
```diff
@@ -29,3 +29,75 @@ http://thtg.vn:8001/thtg.m3u8
...
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="" group-title="",Vajtswv Txojlus
+https://livefta.malimarcdn.com/ftaedge00/vajtswv.stream/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="" group-title="",HmongUSA TV
+https://livefta.malimarcdn.com/ftaedge00/hmongusatv.sdp/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="" group-title="",Hmong Star TV
+https://livefta.malimarcdn.com/ftaedge00/hmongstartv.sdp/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="" group-title="",KhmerTV
+https://livefta.malimarcdn.com/ftaedge00/khmertv2020.stream/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="" group-title="",Tea TV 2
+https://livefta.malimarcdn.com/ftaedge00/teatv2.sdp/playlist.m3u8
```
Comment (LaneSh4d0w) [inline channels/vn.m3u:103]: Language of this channel is not only Vietnamese, I see some English parts.
vlcsnap-2021-05-31-17h35m40s475

File: channels/vn.m3u [hunk 6]
```diff
@@ -29,3 +29,75 @@ http://thtg.vn:8001/thtg.m3u8
...
+https://livefta.malimarcdn.com/ftaedge00/nat.sdp/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="" group-title="",Lao SV TV
+https://livefta.malimarcdn.com/ftaedge00/laosvtv.stream/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="" group-title="",Vajtswv Txojlus
+https://livefta.malimarcdn.com/ftaedge00/vajtswv.stream/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="" group-title="",HmongUSA TV
+https://livefta.malimarcdn.com/ftaedge00/hmongusatv.sdp/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="" group-title="",Hmong Star TV
+https://livefta.malimarcdn.com/ftaedge00/hmongstartv.sdp/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="" group-title="",KhmerTV
```
Comment (LaneSh4d0w) [inline channels/vn.m3u:100]: Language of this channel might be Khmer, to verify,

File: channels/vn.m3u [hunk 7]
```diff
@@ -29,3 +29,75 @@ http://thtg.vn:8001/thtg.m3u8
...
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="" group-title="",ISTV
+https://livefta.malimarcdn.com/ftaedge00/ichannel.sdp/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="" group-title="",NAT TV
+https://livefta.malimarcdn.com/ftaedge00/nat.sdp/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="" group-title="",Lao SV TV
+https://livefta.malimarcdn.com/ftaedge00/laosvtv.stream/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="" group-title="",Vajtswv Txojlus
+https://livefta.malimarcdn.com/ftaedge00/vajtswv.stream/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="" group-title="",HmongUSA TV
+https://livefta.malimarcdn.com/ftaedge00/hmongusatv.sdp/playlist.m3u8
```
Comment (LaneSh4d0w) [inline channels/vn.m3u:97]: Language might be Hmong.

File: channels/vn.m3u [hunk 8]
```diff
@@ -29,3 +29,75 @@ http://thtg.vn:8001/thtg.m3u8
...
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="" group-title="",Hmong Star TV
+https://livefta.malimarcdn.com/ftaedge00/hmongstartv.sdp/playlist.m3u8
```
Comment (LaneSh4d0w) [inline channels/vn.m3u:99]: Language might be Hmong.

**Review by BellezaEmporium — CHANGES_REQUESTED** [review]

**Review by BellezaEmporium — APPROVED** [review]

**Pull request merged**